ql/src/java/org/apache/hadoop/hive/ql/metadata/HiveStorageHandler.java

```diff
@@ -286,6 +286,15 @@ default boolean canSetColStatistics(org.apache.hadoop.hive.ql.metadata.Table tbl
     return false;
   }
 
+  /**
+   *
+   * Check if the storage handler supports load data .
+   * @param parameters table properties
+   * @return StorageFormatDescriptor if the storage handler can support load data
+   */
+  default StorageFormatDescriptor supportsLoadData(Table parameters) {
```

Comment:
method name is misleading, i would expect boolean to be returned

Reply:
changed